Makefile

```diff
@@ -142,7 +142,10 @@ compile-go-lib: install-go-proto-dependencies install-go-ci-dependencies
 	cd sdk/python && python setup.py build_go_lib
 
 test-go: compile-protos-go
+	pip install feast
+	cd go/internal/test && feast init feature_repo && cd feature_repo && feast apply && feast materialize-incremental $(shell date -u +"%Y-%m-%dT%H:%M:%S") && cd ../../../
 	go test ./...
```

Comment:
I'd recommend splitting this into more lines. Also, each line is executed in isolation, so the `cd ../../../` isn't needed.:
```
    cd go/internal/test && feast init feature_repo 
    cd go/internal/test/feature_repo && feast apply && feast materialize-incremental $(shell date -u +"%Y-%m-%dT%H:%M:%S")
    go test ./...
```

Reply:
Removed all of it